p2p_wallet/Scenes/Main/Crypto/Components/Crypto Accounts/Aggregator/CryptoAccountsAggregator.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import KeyAppKitCore
+import Web3
+import Wormhole
+import BigDecimal
+
+struct CryptoAccountsAggregator: DataAggregator {
+    typealias Input = (
+        solanaAccounts: [RenderableSolanaAccount],
+        ethereumAccounts: [RenderableEthereumAccount]
+    )
+    typealias Output = (
+        transfers: [any RenderableAccount],
+        primary: [any RenderableAccount],
+        secondary: [any RenderableAccount]
+    )
+    
+    func transform(input: Input) -> Output {
+        let (solanaAccounts, allEthereumAccounts) = input
+        
+        /// Claimable transfer accounts
```

Comment:
The comments for code need to be started with two slash symbol (//), not three (///)